docs/body.md

```diff
@@ -538,6 +541,19 @@ changes will still be caught without this option enabled, it is
 recommended to leave notify_on_state_change disabled if your
 OpenStack environment is very large.
 
+#### Network and Router Info Events
+
+The OpenStack Infrastructure ZP pulls neutron events for Networks and Routers. However,
+if Neutron is not configured properly to send those events, they cannot be retrieved.
+If you are missing Info events, checks your OpenStack environment's 
+/etc/neutron/neutron.conf. It should have the following:
+
+```
+[oslo_messaging_notifications]
```

Comment:
That would also be fine.  Just want to make sure it's hard to miss, and also concise.. having it in two places seems like it could be confusing.   (3, if you count openstack_amqp_init.py, but that should really only be about the settings that are specific to that device in zenoss, not environmental/setup things)